crates/observe/src/distributed_tracing/tracing_axum.rs

```diff
@@ -29,16 +16,22 @@ pub fn record_trace_id<B>(request: Request<B>) -> Request<B> {
     request
 }
 
+/// Trace context propagation: associate the current span with the OTel trace of
+/// the given request, if any and valid.
 pub fn make_span<B>(request: &Request<B>) -> Span {
-    let headers = request.headers();
-    let uri = request.uri();
-    let method = request.method();
+    let parent_context = global::get_text_map_propagator(|propagator| {
+        propagator.extract(&HeaderExtractor(request.headers()))
+    });
+    let request_id = request_id(request.headers());
 
-    info_span!(
-        "incoming request",
-        ?headers,
-        ?uri,
-        ?method,
+    let span = info_span!(
+        "http_request",
+        uri = ?request.uri(),
```

Comment:
Are these not getting printed on all the child logs?

Reply:
We can easily have both, no?
Unless I miss something we just have to create the span with only the request id and simple emit a separate log with the other info (method, uri, etc.) with the span attached.

Reply:
If we emit a log in this function with the new span that's the only place where we'd have to do it, no?